client/_library.sass

```diff
@@ -4,7 +4,9 @@ $main-colour: #D4E6E3 !default
 $header-colour: #9BC4BC !default
 $main-gray: #D2D2D2 !default
 
-// Spacing - TBD if we want this
+// Sizing
+$small-phone: 481px
+$max-width: 925px
```

Comment:
Where did these values come from? Are they general phone/screen values?

Reply:
480px is supposedly a small phone

925px is an arbitrary size I decided to set the max width to. it was close to what 70% was doing on my smaller screen